scripts/docker/Dockerfile.sycl

```diff
@@ -30,8 +30,8 @@ RUN CMAKE_KEY=2D2CEF1034921684 && \
 ENV PATH=${CMAKE_DIR}/bin:$PATH
 
 ENV SYCL_DIR=/opt/sycl
-RUN SYCL_VERSION=20201119 && \
-    SYCL_URL=https://github.com/intel/llvm/archive && \
+RUN SYCL_VERSION=2020-12 && \
+    SYCL_URL=https://github.com/intel/llvm/archive/ && \
```

Comment:
Why did you add a trailing slash?

Reply:
I just copied from my local configuration and it must have sneaked in there. I can change if you feel strongly about it.

Reply:
Drop it plz